gordo/__init__.py

```diff
@@ -1,8 +1,12 @@
 import logging
-import os
 import traceback
 from typing import Tuple
 import warnings
+from pkgutil import extend_path
+
+
+__path__ = extend_path(__path__, __name__)  # type: ignore
```

Comment:
Please add a comment on why do we need this here

Reply:
fixed